### PR TITLE
Added support of XSLT 2.0

### PR DESCRIPTION
**Pull request by dstenger**
This PR adds the support of XSLT 2.0 for the transformation to HTML of GetFeatureInfo responses as well as GetCapabilities and Exceptions.

Furthermore, the xalan dependency was removed and xerces updated.

### CODE REVIEW

**Comment by copierrj** [comment]
It looks like this PR is also changing the default XSLT version. This would result in uses having to update their existing XSLT documents by adding a version="1.0" attribute.

File: deegree-core/deegree-core-base/pom.xml [hunk 1]
```diff
@@ -38,10 +38,6 @@
       <groupId>commons-codec</groupId>
       <artifactId>commons-codec</artifactId>
     </dependency>
-    <dependency>
```
Comment (copierrj) [inline deegree-core/deegree-core-base/pom.xml]: Why removing this specific xerces dependency?

**Comment by lgoltz** [comment]
> It looks like this PR is also changing the default XSLT version. This would result in uses having to update their existing XSLT documents by adding a version="1.0" attribute. Breaking existing workspaces should be avoided.

The version attribute is required since 1.0 (see https://www.w3.org/1999/11/xslt10.dtd). Saxon is more strict then the the default JAXP implementation and fails if the xslt file is not valid. I think we shouldn't allow invalid xslt files (as before).

File: deegree-core/deegree-core-base/pom.xml (hunk 1 above)
Comment (lgoltz) [reply deegree-core/deegree-core-base/pom.xml]: xerces is used only in deegree-core-commons.

**Comment by tfr42** [comment]
You may need to set a environment variable (system property) as documented in #1110.

**Pull request merged**